plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -504,12 +504,21 @@ function OPDSBrowser:genItemTableFromCatalog(catalog, item_url, username, passwo
                             title = link.title,
                         })
                     elseif link.rel == self.stream_rel then
+                        -- This for loop iterates through all keys in a
+                        --   Entry and looks for the count tag, then stores
+                        --   That key to use for updating the table value.
+                        local count_key = ""
+                        for k, v in pairs(link) do
+                            if string.find(k, ".:count") then
+                                count_key = k
+                            end
```

Comment:
Break the loop when the key is found?

Reply:
That's a good idea! If I make that change, will I have to make a new PR? This is my first time making any open source contribution.

Reply:
Nevermind, I figured it out!